colossalai/kernel/triton/ops.py

```diff
@@ -0,0 +1,204 @@
+import torch
+from torch import nn
+
+import triton
```

Comment:
We should update requirements or check whether triton is installed here.

Reply:
fixed it.